lib/taiko.js

```diff
@@ -2323,12 +2323,21 @@ module.exports.into = (value) => value;
  *
  * @param {string} element - Element/condition to wait for
  * @param {number|time} time - Time to wait. default to 10s
+ * @param { options.message } - Custom message
  * @return {promise}
  */
 
-const waitFor = async (element, time) => {
+const waitFor = async (element, time, options = {}) => {
   validate();
-  let timeout = time || defaultConfig.retryTimeout;
+  let timeout;
+  let message;
+  if (isObject(time)) {
```

Comment:
`waitFor(5000,{"message":"Timed out"})` seems this case is not covered

Reply:
Resolving as we won't encounter this..since wait with only time simply does an explicit time wait